### ISSUE TITLE
Position changes from secondary monitor if primary monitor is disconnected and reconnected

### ISSUE THREAD

**Issue by talboticz** [valinet/ExplorerPatcher#3060]
I have 2 monitors. I use ExplorerPatcher on laptop, used as secondary monitor, docked on left side of screen. I use external monitor as primary display. E.g. ExplorerPatcher is not displayed on primary/external display. When I disconnect external/primary display and reconnect it, ExplorerPatcher is moved from laptop (which I use as secondary display) to exterlan/primary display. At least, the configuration of monitors should be configurable. It could remember ExplorerPatcher position for specific display configuration.

I fight with this issue everytime when I take my laptop from work to home and back.

**Comment by pyrates999** [comment]
I think it's because the moment you disconnect that second monitor that you've set as your primary monitor, windows changes your laptop's monitor to be your primary monitor.

**Comment by talboticz** [comment]
@pyrates999 exactly. But I want to remember my previous configuration, because I travel with my notebook. And that's the point. It's very anoyng if you use 3 or 4 different configurations and everytime you move from one office to another you have to change/modify your configuration.

**Comment by pyrates999** [comment]
In that case, I don't think this is something that EP can fix.  This would be for windows to fix.
